### ISSUE TITLE
Add possibility to count strongholds to the system (payout currently affected)

### ISSUE THREAD

**Issue by enbewu** [ceari/whyattend#38]
I propose to add strongholds to the system which currently would affect only payouts. For the time being it would be great to input the amount of earned points in the given time interval and add an option to add a point value to each point. Most probably this would be some fraction number, i.e. resources/inputed value

PS. I finally got more time and got the system running on my local machine, I'll try to add next tickets with real-life previews. Hope this helps mate.

**Comment by ceari** [comment]
Merged into master. Needs a database upgrade (`alembic upgrade head`) if not done yet.

Have any of you used this feature after my bugfix in November or did it not work?

**Comment by enbewu** [comment]
Checking this right now, but I also were afb for a long time, so give me a while :)

**Comment by enbewu** [comment]
ATM all seems to be working, the only thing that would be good to have is a distinction on the list that it's not a regular CW battle

**Comment by ceari** [comment]
It should say "Stronghold" in the Type column of the battles list.

**Comment by enbewu** [comment]
Nope, it doesn't. Either I'm blind, did something wrong or it doesn't :)

**Comment by ceari** [comment]
Hm strange, it seems the "battleType" in the replay changed from 10 to 11 for Strongholds since November or I somehow did not test with the code I committed back then .. Oh well, it should work now.

Quite annoying to test without replays from my own clan by the way, I have to disable all validation for the clan, players etc. :(

**Comment by ceari** [comment]
Actually, there are "skirmishes" = 10 and "battles for stronghold" = 11 and only the skirmish replays contain an amount of resources earned. Back when I played there was no "battle for stronghold". Does it make sense to track stronghold battles when we can only get the resources for skirmishes?

**Comment by enbewu** [comment]
That's interesting. Battle for stronghold is actually the more important one as the clan gets the most of it.

I understand that you refrain from playing currently (can't blame you :) so - the battle for stronghold looks like a ladder for a landing province. Clans fight for each building that is on their path, sometimes fighting couple of times for the same building - it depends on their success rate. A typicall good clan has at most 3-4 battles for stronghold. Therefore it makes sense that there are no resources in the replay as it's calculated after the whole battle ends.

From here I see two options, with one common issue: detect both types of battles, add description 10 = "Skirmish", 11 = "Stronghold"

1) Scenario - try to sum the resources from the skirmishes, and option to update earned resources per player in the Payout screen
2) Ditch the resources completely and just add strongold+skirmish battles to player activity.

In fact, I just came up with the idea, that in the second option we might just split the activities: one for CW and other for Strongholds and duplicate the system of points for being in the battle like in CW.

**Comment by ceari** [comment]
Both options are possible, I think the second one requires quite some changes overall though.

Unfortunately I can't promise when I start to work on this - I'm really busy with work again and not in the mood to work on side projects, sorry. Next weekend might be better.
